homeassistant/components/deconz/deconz_device.py

```diff
@@ -117,11 +124,15 @@ class DeconzSceneMixin(DeconzDevice):
 
     _device: PydeconzScene
 
-    def __init__(self, device: PydeconzScene, gateway: DeconzGateway) -> None:
+    def __init__(
+        self,
+        device: PydeconzScene,
+        gateway: DeconzGateway,
+    ) -> None:
         """Set up a scene."""
         super().__init__(device, gateway)
 
-        self._attr_name = device.full_name
+        self._attr_name = self._device.full_name
```

Comment:
I'm not sure if this change is required.
```suggestion
        self._attr_name = device.full_name
```

Reply:
Done